Cargo.toml

```diff
@@ -22,6 +22,9 @@ include = [
 [dependencies]
 byteorder = "1.3"
 errno = "0.2"
+fern = { version = "0.5", features = ["colored"], optional = true }
+# statically turn off logging for wasmer by default, fetaures override this
```

Comment:
```suggestion
# statically turn off logging for wasmer by default, features override this
```

Reply:
I deleted this comment because it's no longer true!